.github/workflows/netbench.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - uses: actions-rs/toolchain@v1.0.7
         id: toolchain
         with:
-          toolchain: 1.63.0
+          toolchain: 1.64.0
```

Comment:
If we are pinning `winnow` to keep the MSRV at 1.63, don't we want to keep testing on 1.63?

Reply:
netbench's MSRV is less conservative than s2n-quic's

Reply:
Ahh, missed the path in the fail.